docs/sources/configure-client/grafana-agent/_index.md

```diff
@@ -1,33 +1,41 @@
 ---
-title: "Grafana Alloy and Grafana Agent"
-menuTitle: "Grafana Alloy and Grafana Agent"
-description: "Send data from your application via using Grafana Alloy or Grafana Agent."
+title: "Grafana Alloy"
+menuTitle: "Grafana Alloy"
+description: "Send data from your application via using Grafana Alloy."
 weight: 10
 aliases:
   - /docs/phlare/latest/configure-client/grafana-agent/
 ---
 
-# Grafana Alloy and Grafana Agent
+# Grafana Alloy
 
 You can send data from your application using Grafana Alloy (preferred) or Grafana Agent (legacy) collectors.
 Both collectors support profiling with eBPF, Java, and Golang in pull mode.
 
-[Grafana Alloy](https://grafana.com/docs/alloy/latest/) is a vendor-neutral distribution of the OpenTelemetry (OTel) Collector.
+[Grafana Alloy](https://grafana.com/docs/alloy/<ALLOY_VERSION>/) is a vendor-neutral distribution of the OpenTelemetry (OTel) Collector.
 Alloy uniquely combines the very best OSS observability signals in the community.
```

Comment:
One minor suggestion: should we consider standardizing the usage of `Grafana Alloy` vs. `Alloy` throughout the docs for consistency?

Reply:
Within the Alloy docs we use a variable to manage this... and the first mention of Alloy uses `Grafana Alloy` and all subsequent mentions within a single topic are just  `Alloy`

Reply:
I should check through these docs and make sure we're consistent.

Reply:
Done.